packages/pubsub/src/topic.js

```diff
@@ -456,6 +456,10 @@ Topic.prototype.publish = function(messages, options, callback) {
     method: 'publish',
   };
 
+  if (options && is.number(options.timeout)) {
```

Comment:
Just if `is.number` works here, because we know `options` is defaulted to an object above.